exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -392,10 +419,12 @@ drill.exec.options:  {
     exec.query_profile.debug_mode: false,
     exec.query_profile.save: true,
     exec.queue.enable: false,
-    exec.queue.large: 10,
-    exec.queue.small: 100,
+    exec.queue.large: 2,
```

Comment:
Do we need to add description of this properties in `drill-override-example.conf`?
Also is there a reason why we made limit smaller than before?